### PR TITLE
l10n: uk: update translation

### PR DESCRIPTION
**Pull request by arkid15r**

### CODE REVIEW

File: po/uk.po [hunk 1]
```diff
@@ -2370,33 +2383,39 @@ msgid ""
...
 msgid "--chmod param '%s' must be either -x or +x"
-msgstr ""
+msgstr "--chmod параметр \"%s\" має бути -x або +x"
 
 #: builtin/add.c:518 builtin/checkout.c:1809 builtin/commit.c:374
 #: builtin/reset.c:364 builtin/rm.c:279 builtin/stash.c:1772
 #, c-format
 msgid "'%s' and pathspec arguments cannot be used together"
-msgstr "'%s' та аргументи специфікації шляху не можна використовувати разом"
+msgstr "\"%s\" та аргументи визничника шляху не можна використовувати разом"
```
Comment (kasya) [inline po/uk.po]:
```suggestion
msgstr "\"%s\" та аргументи визначника шляху не можна використовувати разом"
```

File: po/uk.po [hunk 2]
```diff
@@ -2098,14 +2098,14 @@ msgid ""
...
-"Існує декілька віддалених призначень, специфікатори отримування посилань "
+"Існує декілька віддалених призначень, визничники отримування посилань "
 "яких розвʼязуються у віддалено \n"
 "відстежуване посилання \"%s\":\n"
 "%s\n"
 "Зазвичай це помилка конфігурації.\n"
 "\n"
 "Щоб підтримувати налаштування гілок відстежування, переконайтеся, що\n"
-"специфікатори отримування посилань різних віддалених призначень "
+"визничники отримування посилань різних віддалених призначень "
```
Comment (kasya) [inline po/uk.po]:
```suggestion
"визначники отримування посилань різних віддалених призначень "
```

File: po/uk.po [hunk 3]
```diff
@@ -2098,14 +2098,14 @@ msgid ""
 "different remotes' fetch refspecs map into different\n"
 "tracking namespaces."
 msgstr ""
-"Існує декілька віддалених призначень, специфікатори отримування посилань "
+"Існує декілька віддалених призначень, визничники отримування посилань "
```
Comment (kasya) [inline po/uk.po]:
```suggestion
"Існує декілька віддалених призначень, визначники отримування посилань "
```

File: po/uk.po [hunk 4]
```diff
@@ -2635,6 +2667,7 @@ msgstr "Латання не вдалося на %s %.*s"
 #: builtin/am.c:1885
 msgid "Use 'git am --show-current-patch=diff' to see the failed patch"
 msgstr ""
+"Використовуйте git am --show-current-patch=diff, щоб побачити невдалу латку"
```
Comment (kasya) [inline po/uk.po]:
```suggestion
"Використовуйте \"git am --show-current-patch=diff\", щоб побачити невдалу латку"
```

File: po/uk.po [hunk 5]
```diff
@@ -2646,6 +2679,9 @@ msgid ""
 "If there is nothing left to stage, chances are that something else\n"
 "already introduced the same changes; you might want to skip this patch."
 msgstr ""
+"Без змін - ви забули використати \"git add\"?\n"
+"Якщо нічого  індексувати, є ймовірність того, що хтось інший\n"
```
Comment (kasya) [inline po/uk.po]:
```suggestion
"Якщо нічого індексувати, є ймовірність того, що хтось інший\n"
```

File: po/uk.po [hunk 6]
```diff
@@ -2693,70 +2735,71 @@ msgstr "git am [<опції>] [(<скринька> [<поштова директ
...
 #: builtin/am.c:2333
 msgid "allow fall back on 3way merging if needed"
-msgstr ""
+msgstr "дозволити повернутися до тристороннього злиття у разі потреби"
 
 #: builtin/am.c:2334 builtin/init-db.c:555 builtin/prune-packed.c:17
 #: builtin/repack.c:786 builtin/stash.c:1010
 msgid "be quiet"
-msgstr ""
+msgstr "працювати тихесенько"
```
Comment (kasya) [inline po/uk.po:2759]: 🔥

File: po/uk.po [hunk 7]
```diff
@@ -4723,7 +4774,7 @@ msgstr "використовувати їхню версію для незлит
 
 #: builtin/checkout.c:1640
 msgid "do not limit pathspecs to sparse entries only"
-msgstr "не обмежувати специфікації шляхів лише розрідженими записами"
+msgstr "не обмежувати визничники шляхів лише розрідженими записами"
```
Comment (kasya) [inline po/uk.po]:
```suggestion
msgstr "не обмежувати визначники шляхів лише розрідженими записами"
```

File: po/uk.po [hunk 8]
```diff
@@ -6977,7 +7028,7 @@ msgstr "Файл блокування створено, але не звітов
 
 #: builtin/fetch.c:40
 msgid "git fetch [<options>] [<repository> [<refspec>...]]"
-msgstr "git fetch [<опції>] [<сховище> [<специфікатор посилання>...]]"
+msgstr "git fetch [<опції>] [<сховище> [<визничник посилання>...]]"
```
Comment (kasya) [inline po/uk.po]:
```suggestion
msgstr "git fetch [<опції>] [<сховище> [<визначник посилання>...]]"
```

File: po/uk.po [hunk 9]
```diff
@@ -7349,7 +7400,7 @@ msgstr "fetch --all не приймає сховище як аргумент"
 
 #: builtin/fetch.c:2234
 msgid "fetch --all does not make sense with refspecs"
-msgstr "fetch --all не має сенсу зі специфікаторами посилань"
+msgstr "fetch --all не має сенсу з визничниками посилань"
```
Comment (kasya) [inline po/uk.po]:
```suggestion
msgstr "fetch --all не має сенсу з визначниками посилань"
```

File: po/uk.po [hunk 10]
```diff
@@ -11550,7 +11601,7 @@ msgstr "неможливо виконати видалення в precious-objec
 
 #: builtin/pull.c:69
 msgid "git pull [<options>] [<repository> [<refspec>...]]"
-msgstr "git pull [<опції>] [<сховище> [<специфікатор-посилання>...]]"
+msgstr "git pull [<опції>] [<сховище> [<визначник-посилання>...]]"
```
Comment (kasya) [inline po/uk.po]:
```suggestion
msgstr "git pull [<опції>] [<сховище> [<визначник посилання>...]]"
```

**Review by kasya — COMMENTED** [review]
LGTM!

File: po/uk.po (hunk 3 above)
Comment (arkid15r) [reply po/uk.po]: 😮

File: po/uk.po (hunk 1 above)
Comment (arkid15r) [reply po/uk.po]: 😮

**Pull request merged**